testing/py-html5lib/APKBUILD

```diff
@@ -1,41 +1,55 @@
 # Contributor: Fabian Affolter <fabian@affolter-engineering.ch>
+# Contributor: Stefan Wagner <stw@bit-strickerei.de>
 # Maintainer: Fabian Affolter <fabian@affolter-engineering.ch>
 pkgname=py-html5lib
 _pkgname=html5lib
-pkgver=0.999
+pkgver=1.0.1
 pkgrel=0
 pkgdesc="A Python HTML parser"
 url="https://github.com/html5lib/html5lib-python"
 arch="noarch"
 license="MIT"
-depends="python2"
-depends_dev=""
-makedepends="python2-dev py-setuptools"
-install=""
-subpackages=""
+depends="py-webencodings"
+checkdepends="pytest pytest-expect py-mock"
+makedepends="python2-dev python3-dev py-setuptools"
+subpackages="py2-$_pkgname:_py2 py3-$_pkgname:_py3"
 source="https://files.pythonhosted.org/packages/source/${_pkgname:0:1}/$_pkgname/$_pkgname-$pkgver.tar.gz"
+builddir="$srcdir"/$_pkgname-$pkgver
 
-_builddir="$srcdir"/$_pkgname-$pkgver
-prepare() {
-	local i
-	cd "$_builddir"
-	for i in $source; do
-		case $i in
-		*.patch) msg $i; patch -p1 -i "$srcdir"/$i || return 1;;
-		esac
-	done
+build() {
+	cd "$builddir"
+	python2 setup.py build
+	python3 setup.py build
 }
 
-build() {
-	cd "$_builddir"
-	python2 setup.py build || return 1
+check() {
+	py.test-2
```

Comment:
needs add `cd "$builddir"`

Reply:
https://wiki.alpinelinux.org/wiki/APKBUILD_examples:Python#Multiversion_package_without_executables

Reply:
Added $builddir. Thanks for dropping in this link. I did some more changes according to the documentation.